src/lib.rs

```diff
@@ -17,6 +17,7 @@ extern crate reqwest;
 extern crate rpki;
```

Comment:
I don’t think you need any of the `extern crate` things any more.

Reply:
This was cleaned up. The `#[macro_use]` instances were kept for now, as removing them would require a lot of fixes to use statements. So, it's better to do this when there are no big open PRs.